cmake/dependencies.cmake

```diff
@@ -43,9 +43,15 @@ set_target_properties(RapidJSON::rapidjson PROPERTIES
 )
 
 ##########################
-#           pq           #
+#         libpq          #
```

Comment:
other similar names omit `lib` prefix

Reply:
Well, it depends on how they are referenced in the source. [Here](https://www.postgresql.org/docs/9.5/libpq.html) they reference it as libpq, while [TBB](https://github.com/intel/tbb) is referenced without `lib` prefix in the source.